Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.10-slim
+
+RUN apt update && apt install -y \
```

Comment:
I wonder if this could be cleaned up a bit by putting this in a separate file and then in that file we could use some bash magic to make it a little cleaner.